core/field_multilineinput.js

```diff
@@ -170,6 +210,15 @@ Blockly.FieldMultilineInput.prototype.render_ = function() {
     y += lineHeight;
   }
 
+  if (this.isBeingEdited_) {
+    var htmlInput = /** @type {!HTMLElement} */(this.htmlInput_);
+    if (this.isOverflowedY_) {
+      Blockly.utils.dom.addClass(htmlInput, 'blocklyHtmlTextAreaInputOverflowedY');
+    } else {
+      Blockly.utils.dom.removeClass(htmlInput, 'blocklyHtmlTextAreaInputOverflowedY');
+    }
+  }
+
```

Comment:
So I should probably explain why I have decided enabling the scrollbar manually like this.

Initially I tried using `overflow-y:auto;`, but because the textarea is not resizable, I would get a glitch like this:
![glitch](https://user-images.githubusercontent.com/42584490/101525012-c56e8200-399b-11eb-9bcd-a26a41d0e825.gif)

As I understand it, when we have a non-resizable textarea filled with text, another symbol automatically triggers scrollbar, and even though the next moment we set the correct width, the scrollbars' width is still there and it still counts as existing. Probably has something to do with the reflow order... and so we get the phantom scrollbar space. I tried fixing this thing, but eventually it got too painful, and I settled with this approach.